File: Daily_Questions/Strings/2486.cpp
// https://leetcode.com/problems/append-characters-to-string-to-make-subsequence/description/

// class Solution
// {
//     public:
//         int appendCharacters(string s, string t)
//         {
//             int i = 0, j = 0;
//             int n = s.size(), m = t.size();
//             int result = 0;
//             while (i < n && j < m)
//             {
//                 if (s[i] == t[j])
//                 {
//                     // result++;
//                     j++;
//                 }

//                     i++;

//             }
//             return m - j;
//         }
// };

File: Daily_Questions/Strings/2486.java
// //
// https://leetcode.com/problems/append-characters-to-string-to-make-subsequence/description/

// class Solution {

// public int appendCharacters(String s, String t) {
// int i = 0;
// int longestPrefix = 0;

// while (i < s.length() && longestPrefix < t.length()) {
// if (s.charAt(i) == t.charAt(longestPrefix)) {
// longestPrefix++;
// }
// i++;
// }
// return t.length() - longestPrefix;

// }
// }
